aries-site/src/layouts/content/UpdateNotification.js

```diff
@@ -0,0 +1,58 @@
+import { Anchor, Text, Notification } from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+import { ViewContext } from '../../pages/_app';
+import { useContext } from 'react';
+
+export const UpdateNotification = ({ name }) => {
+  const { wholeViewHistory } = useContext(ViewContext) || undefined;
+
+  function InlineNotification({ dateText, message }) {
+    return (
+      <Notification
+        width="large"
+        title={
+          dateText +
+          new Date(wholeViewHistory[name]?.date)
+            .toDateString()
+            .split(' ')
+            .slice(1)
+            .join(' ')
+        }
+        message={message}
+        margin={{ bottom: 'medium' }}
+        icon={<CircleInformation />}
+      />
+    );
+  }
+
+  if (wholeViewHistory && name in wholeViewHistory) {
+    return (
+      <>
+        {wholeViewHistory[name]?.type === 'Update' && (
```

Comment:
Will this be one or the other? Or is it possible that there could be both.

If it's one or the other, I think we could consolidate the logic a bit. Something like:

```
<InlineNotification
   dateText={wholeViewHistory[name]?.type === 'Update' ? "Updated " : "Added on "
   ...
/>